packages/account-v2/src/pages/ConnectedApps/ConnectedAppsResponsive.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { formatDate, getFormattedAppScopes } from '../../utils/connectedAppsUtils';
+import { OauthApps } from '@deriv/api-types';
+import { ConnectedAppsResponsiveColumn } from './ConnectedAppsResponsiveColumn';
+import { Button } from '@deriv-com/ui';
+
+type TConnectedAppsResponsive = {
+    connectedApps: OauthApps;
+    handleToggleModal: (appId: number) => void;
+};
+
+export const ConnectedAppsResponsive = ({ connectedApps, handleToggleModal }: TConnectedAppsResponsive) => (
+    <div className='flex flex-col gap-8'>
+        {connectedApps.map(connectedApp => (
+            <div className='bg-solid-grey-2 rounded-default p-16' key={connectedApp.app_id}>
+                <div className='flex flex-row justify-between gap-16'>
```

Comment:
just to confirm, do we need to set flex as row? is it so by default?

Reply:
Thanks for pointing this out, default works just fine 🙏